python/examples/tutorials/Data_Validation_Tutorial.ipynb

```diff
@@ -0,0 +1,4235 @@
+        }
+      },
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "![image.png](attachment:image.png)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lSIwRzq0liRI"
+      },
+      "source": [
+        "- A whylogs __profile__ contains a set of metrics that summarize the original data. \n",
+        "- __Metric Constraints__ validate __Metrics__ present on the __Profile__ to generate a Report. This report will tell us whether the data meets the data quality constraints we defined.\n",
+        "- Those metrics can be standard metrics, such as Distribution, Frequent Items, or Types metrics. They can also be __Condition Count Metrics__.\n",
+        "- Condition Count Metrics count the number of occurrences a certain relation passed/failed. For example, the number of times the rows for the column `Name` was equal to `Bob`\n",
+        "- For a profile to contain Condition Count Metrics, you have first to specify a Condition __before__ logging your data, and pass that to your Dataset Schema.\n",
+        "- The Dataset Schema configures the behavior for tracking metrics in whylogs.\n"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "y3vXIHF4lyIi"
+      },
+      "source": [
+        "## Defining the Conditions\n",
+        "\n",
+        "The conditions are defined through a whylogs' `Condition` object. There are several different ways of assembling a condition. In the following example, we will define a custom function that will verify if the string is properly formatted as a date. We'll also define a second relation that will validate the text against a defined regex pattern to verify the url formatting.\n",
+        "\n",
+        "Since we can define multiple conditions for a single column, we'll assemble the conditions into dictionaries, where the key is the condition name. Each dictionary will be later attached to the relevant column."
```

Comment:
```suggestion
        "Since we can define multiple conditions for a single column, we'll assemble the conditions into dictionaries, where the key is the condition name. Each dictionary will later be attached to the relevant column."
```